.cicd/base-images-pinned.yml

```diff
@@ -84,7 +84,7 @@ steps:
     agents: "queue=mac-anka-node-fleet"
     timeout: 180
     skip: ${SKIP_MACOS_10_15}${SKIP_MAC}
-    
+
   - label: ":ubuntu: Ubuntu 16.04 - Base Image Pinned"
```

Comment:
I noticed there is no unpinned ubuntu 16.04. Is this a bug?

Reply:
Are you still working off release/2.1.x? I don't know why no unpinned on ubuntu 16.04. It is too old. No need to fix it.